tests/ui/src/utilities/globalSetup.ts

```diff
@@ -13,9 +13,16 @@ async function globalSetup(config: FullConfig) {
     const workspace = process.env.WORKSPACE as string;
     const env = process.env.ENV as string;
     const region = process.env.REGION as string;
-    const distribution = getCloudFrontUrl(region, env, workspace);
-    const baseUrl = JSON.parse(distribution).DomainName;
-    process.env.baseUrl = baseUrl;
+    await getCloudFrontUrl(region, env, workspace).then((distribution) =>{
+      if (distribution) {
+        const baseUrl = distribution.DomainName;
+        process.env.baseUrl = baseUrl;
+      } else {
+        console.log('No domain name found');
+      }
+    }).catch((error) => {
+      console.error('Error getting cloud front url', error);
+    });
```

Comment:
You're doing all your error handling in the function above, so should be able to simply this with just

```ts
const distribution = await getCloudFrontUrl(region, env, workspace);
if (distribution) {
    process.env.baseUrl = distribution.DomainName;
} else {
    console.log('No domain name found');
}
```